bin/dotd

```diff
@@ -152,6 +152,28 @@ def should_i(question)
   end
 end
 
+def should_i_really(question, confirmation)
+  loop do
+    print "Should I really #{question}? If so, type #{underline confirmation}: "
+    response = flush_and_gets.strip
+    puts ''
+    if response == confirmation
```

Comment:
Just to force the dotd to be very deliberate about confirming this step. I suppose it's highly unlikely that anyone would type `dtp` without intending to deploy, but having to match the case just makes you a little slower and hopefully a little more careful.